classes/forms.php

```diff
@@ -482,15 +482,19 @@ protected static function add_details( $forms ){
 	 */
 	public static function save_form( $data, $type = 'primary' ){
 
-		// option value labels
+		
 		if(!empty($data['fields'])){
 			foreach($data['fields'] as &$field){
+				// option value labels
 				if(!empty($field['config']['option']) && is_array($field['config']['option'])){
 					foreach($field['config']['option'] as &$option){
 						if(!isset($option['value'])){
 							$option['value'] = $option['label'];
 						}
 					}
+				// trim manual calculations
+				} else if(!empty($field["config"]["manual_formula"]) && is_string($field["config"]["manual_formula"])){
+					$field["config"]["manual_formula"] = str_replace( "\r", '', str_replace( "\n", '', str_replace( ' ', '', trim( $field["config"]["manual_formula"] ) ) ) );
```

Comment:
@New0 This is a lot, without a test.

Can you please add a method to `Caldera_Forms_Santize` class to do this. And test against possible inputs, so we know it is safe?

Also what about `$str = trim( str_replace( '<br>', '', nl2br($str)  ));` nl2br should convert all line breaks to html br tags, which we can then lift out.